content/docs/ops/runbook/troubleshooting-bosh.md

```diff
@@ -83,69 +83,51 @@ the Troubleshooting Bosh Managed VMS section below.
 In order to troubleshoot a Bosh-managed VMS, you will need to first access a
 jumpbox, outlined previously.
 
-### Setting the deployment
+### Finding the deployment
 
-You'll need to set up `bosh` to the correct deployment. You can verify that the
-deployment is set by running the following command:
-
-```sh
-$ bosh status
-
-# ... output shortened for brevity
-
-Deployment
-  not set
-```
-
-You can get a list of all the current deployments by running the following
-command:
+In the pre-v2 BOSH CLI, you needed to set your deployment.
+In BOSH CLI v2, you just use the name of the deployment in all the commands you use.
+In order see all the deployments, run the following command:
 
 ```sh
 $ bosh deployments
-+----------+-------------------+-------------------------+
-| Name     | Release(s)        | Stemcell(s)             |
-+----------+-------------------+-------------------------+
-| admin-ui | admin-ui/10       | bosh-aws-xen/3232.11    |
+Name                   Release(s)                           Stemcell(s)                                      Team(s)  Cloud Config
+admin-ui               clamav/9                             bosh-aws-xen-hvm-ubuntu-trusty-go_agent/3421.11  -        latest
 
 # ... output shortened for brevity
 
-Deployments total: X
+X deployments
+
+Succeeded
 ```
 
 After selecting a deployment from the named list from the above's output, you
 will need to download the manifest for that deployment to the local file-system.
 
 ```sh
-$ bosh download manifest <DEPLOYMENT_NAME> ./<DEPLOYMENT_NAME>.yml
-$ bosh deployment ./<DEPLOYMENT_NAME>.yml
-$ bosh status
-
-# ... output shortened for brevity
-
-Deployment
-  Manifest: /path/to/<DEPLOYMENT_NAME>.yml
+$ bosh -d <DEPLOYMENT_NAME manifest > ./<DEPLOYMENT_NAME>.yml
```

Comment:
Should probably be `bosh -d <DEPLOYMENT_NAME> manifest > ./<DEPLOYMENT_NAME>.yml